apis/Google.Datastore.V1Beta3/Google.Datastore.V1Beta3.Snippets/DatastoreDbSnippets.cs

```diff
@@ -90,10 +91,11 @@ public void StructuredQuery()
             {
                 Filter = Filter.Equal("author", "Jane Austen")
             };
-            DatastoreQueryResults results = db.RunQuery(query);
-            // DatastoreQueryResults implements IEnumerable<Entity>, but you can
-            // call AsEntityResults(), AsBatches() or AsResponses() to see the query
-            // results in whatever way makes most sense for your application.
+            LazyDatastoreQuery results = db.RunQuery(query);
```

Comment:
I think there's a problem with discoverability with these two methods. Users will look for and discover `RunQuery` but won't come across `IterateQuery` next to it in IntelliSense. I would still favor something like RunQuery / RunQueryLazy for proximity.